js/interface.js

```diff
@@ -550,11 +550,22 @@ function uploadFiles(files) {
     files.forEach(function(file) {
       addFile(file);
     });
-
+    sortImageFiles();
     $progress.addClass('hidden');
   });
 }
 
+function sortImageFiles() {
```

Comment:
why does it say "Image" in the name? Any type of file and folder should be sorted.

Note: the order to use should be Folders A-Z and then Files A-Z

e.g.

```
A (folder)
B (folder)
Z (folder)
A (file)
C (file)
c (file)
D (file)

Reply:
Fixed.